cannot import css in `v2.2.0`

greetings!

I am trying to upgrade shikwasa to `v2.2.0` at [here](https://github.com/IvanWoo/subjpop.github.io/pull/34), but I noticed that the css file cannot be imported properly with the following error msg

```js
import "shikwasa/dist/style.css";
import { Chapter, Player } from "shikwasa";

...
```

```sh
...
ERROR in ./src/podcastPlayer/index.js 1:0-33
[30](https://github.com/IvanWoo/subjpop.github.io/actions/runs/4157082134/jobs/7191312599#step:7:31)
Module not found: Error: Package path ./dist/style.css is not exported from package /home/runner/work/subjpop.github.io/subjpop.github.io/node_modules/shikwasa (see exports field in /home/runner/work/subjpop.github.io/subjpop.github.io/node_modules/shikwasa/package.json)
[31](https://github.com/IvanWoo/subjpop.github.io/actions/runs/4157082134/jobs/7191312599#step:7:32)
 @ ./src/index.js 5:0-50 11:2-21
```

seems like it's related to the `exports` field of `package.json`

https://github.com/jessuni/shikwasa/blob/ecf7a3e3e607540b53aa3beb54a366641ea577c3/package.json#L10-L14

I can import the css after removing the `exports` part based on the discussion at [here](https://github.com/vitejs/vite/discussions/2657#discussioncomment-1307789)

I don't mind opening a pr for it, but not sure if it's the right way going forward b/c it might break commonjs import

thanks!

--- a/package.json
+++ b/package.json
@@ -11,6 +11,10 @@
     ".": {
       "import": "./dist/shikwasa.es.js",
       "require": "./dist/shikwasa.cjs.js"
+    },
+    "./dist/style.css": {
+      "import": "./dist/style.css",
+      "require": "./dist/style.css"
     }
   },
   "devDependencies": {

